XSConsoleTask.py

```diff
@@ -53,11 +56,7 @@ def Status(self):
         return status
 
     def Result(self):
-        if self.Completed():
-            result = self.completionStatus
-        else:
-            result= self.session.xenapi.task.get_status(self.hotOpaqueRef.OpaqueRef())
-        return HotOpaqueRef(result, 'any')
```

Comment:
Previously results were wrapped in a `HotOpaqueRef()` and now they're not. Is that a problem?

Reply:
I tested the change, and it works.
I'll double-check it.

Reply:
Although the previous code worked, use `HotOpaqueRef()` to keep consistency, and use `srRef.OpaqueRef()` to get the original result.